lib/components/chat_history/twitch/message.dart

```diff
@@ -116,7 +116,10 @@ Iterable<InlineSpan> parseText(
         ),
       );
     } else {
-      return TextSpan(children: tokenize(element.text, tagStyle).toList());
+      return WidgetSpan(
+        child: RichText(
+            text: TextSpan(children: tokenize(element.text, tagStyle).toList())),
+      );
```

Comment:
I would keep the old code. it seems like the change is introducing weird changes to the font size of a message. It looked like the picture below.
![image](https://user-images.githubusercontent.com/39935368/126049627-1a2b6375-d0c3-48d6-8ce3-53308d9434e6.png)